config/manager/manager.yaml

```diff
@@ -24,7 +24,10 @@ spec:
         - name: LOG_STRUCTURED
           value: 'false'
         - name: LOG_LEVEL
-          value: INFO
+          valueFrom:
+            configMapKeyRef:
+              name: environment-variables
+              key: LOG_LEVEL
```

Comment:
I'm not sure if it makes sense to create a configmap just for this. It feels like changing the value in `deploy/operator.yaml` and applying it is easier and makes more sense. What do you think @tplavcic @inelpandzic?

Reply:
@egegunes I am also agree with you.

Reply:
Yes @egegunes , wanted to say the same thing.

Reply:
`deploy/operator.yaml` is created by using `manifests`rule from Makefile and `deploy` rule is used to run the operator. With the patch single change is needed for all env variables for operator (in case you are going to have many). That way you will have single call `make install deploy` what is more elegant than 3 step process:  `make install` -> change operator files -> `make deploy`.
However,  I'm not sure about the problems that new config map will bring and I'm ok even without this change, just I would suggest to update PR #103 with information about changing env variables.

Reply:
@an3l please update PR https://github.com/percona/percona-server-mysql-operator/pull/103